src/agent/Cargo.toml

```diff
@@ -88,8 +88,14 @@ hex = { version = "0.4.2", optional = true }
 sev = { git = "https://github.com/virtee/sev", version = "1.2", default-features = false, features = ["snp"], optional = true }
 
 # Agent Policy
-# TODO: use regorus version 0.1.5 when it will be released to crates.io.
-regorus = { git = "https://github.com/anakrish/regorus", rev = "fc6f5d4", default-features = false, features = ["arc", "base64", "base64url", "regex"], optional = true }
+regorus = { version = "0.2.6", default-features = false, features = [
+    "arc",
+    "regex",
+    "std",
+    "base64",
+    "base64url",
```

Comment:
We might need to include these regorus features in upstream at some point. 

I needed to add these to keep some existing functionality on rules.rego related to base64 and base64 url. If these features don't get added, I get failures like 
<img width="697" alt="image" src="https://github.com/user-attachments/assets/201ce09d-4e8d-44c0-afbe-3b23150714b8" />

Reply:
Fyi I'm including base64 as part of my confidential storage PR.